src/canvas-extractor/README.md

```diff
@@ -23,37 +48,24 @@ to `.env`, and adjust to your desired parameters. Supported parameters:
 
 | Description | Required | Command Line Argument | Environment Variable |
 | ----------- | -------- | --------------------- | -------------------- |
-| Base Canvas URL | yes | -b or --base-url | CANVAS_BASE_URL |
-| Canvas API access token | yes |  -a or --access-token | CANVAS_ACCESS_TOKEN |
-| Output Directory | no (default: [working directory]/data) | -o or --output-directory | OUTPUT_DIRECTORY |
-| Log level | no (default: INFO) | -l or --log-level | LOG_LEVEL |
-| Start date | yes | -s or --start_date | START_DATE |
-| End date | yes | -e or --end_date | END_DATE |
+| Base Canvas URL | yes | `-b` or `--base-url` | CANVAS_BASE_URL |
+| Canvas API access token | yes | `-a` or `--access-token` | CANVAS_ACCESS_TOKEN |
+| Output Directory | no (default: [working directory]/data) | `-o` or `--output-directory` | OUTPUT_DIRECTORY |
+| Start date*, yyyy-mm-dd format | yes | `-s` or `--start_date` | START_DATE |
+| End date*, yyyy-mm-dd format | yes | `-e` or `--end_date` | END_DATE |
+| Log level** | no (default: INFO) | `-l` or `--log-level` | LOG_LEVEL |
+
+\* _Start Date_ and _End Date_ are used in pulling course and system activity
+data and would typically span a semester or equivalent school calendar timespan.
 
+\** Valid values for the optional _log level_:
 
-Valid log levels:
 * DEBUG
 * INFO(default)
 * WARNING
 * ERROR
 * CRITICAL
 
-## Execution
-
-Execute the extractor with CLI args:
-
-```bash
-poetry run python.exe canvas_extractor -b your-canvas-url -a your-api-token -s start-date-range -e end-date-range
-```
-
-Alternately, run with environment variables or `.env` file:
-
-```bash
-poetry run python.exe canvas_extractor
-```
-
-For detailed help, execute `poetry run python canvas_extractor -h`.
-
 ### Output
 
 CSV files in the data(or the specified output) directory with the LMS UDM format.
```

Comment:
Adding a link to the UDM page in Tech Docs from here could be a good idea.

Reply:
```suggestion
CSV files in the data(or the specified output) directory with the [LMS Unifying Data Model](https://techdocs.ed-fi.org/display/EDFITOOLS/LMS+Unifying+Data+Model) format.
```